package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "ycb-config",
-    "version": "2.3.0-beta.2",
+    "version": "2.3.0-beta.3",
```

Comment:
Are we using the beta anywhere? Should we backport this change to 2.2.x?

Reply:
There are some internal use cases for the beta version currently. Given that the only change of note here is bumping the _minimum_ version of YCB to include the `main` changes, I can raise a PR to backport that to the non-beta version.

I really do want to drop the pre-release ASAP, but it's more involved and will require more planning.

Reply:
https://github.com/yahoo/ycb-config/tree/v2.2.1